node-server/api/location.js

```diff
@@ -42,9 +42,107 @@ router.route('/').get(async (req, res) => {
         })
         const location = await Location.find();
         res.json(location);
-    } catch(e){
+    } catch (e) {
         res.status(400).json("Error: " + e);
     }
-  })
+})
 
-  module.exports = router
\ No newline at end of file
+// POST Route: Creates New Location
+router.route('/addLocation').post(async (req, res) => {
+    try {
+        // Constants
+        const name = req.body.name;
```

Comment:
const { name, description, ... } = req.body

Reply:
USE DECONSTRUCTOR, should be better